### PR TITLE
refactor: drop db migration patch

### PR DESCRIPTION
**Pull request by jsjoeio**
This drops the patch and also the tests (which were flakey and our longest running e2e tests).

- refactor: remove database migration patch & test
- chore: refresh patches

Fixes https://github.com/coder/code-server/issues/5482

### CODE REVIEW

File: patches/telemetry.diff [hunk 1]
```diff
@@ -104,6 +104,13 @@ Index: code-server/lib/vscode/src/vs/workbench/services/telemetry/browser/teleme
 -				sendErrorTelemetry: this.sendErrorTelemetry,
 -			};
 -			this.impl = this._register(new BaseTelemetryService(config, configurationService, productService));
+-
```
Comment (jsjoeio) [inline patches/telemetry.diff:107]: @code-asher I refreshed all the patches. Does this look okay to you?

Comment (code-asher) [reply patches/telemetry.diff:107]: Looks like the blocks changed a little, probably due to using a different diff algorithm, but I think it should be fine!  Looks like I am using `patience` (you can see what you have by running `git config diff.algorithm` in the code-server repo).

**Review by code-asher — APPROVED** [review]

File: patches/telemetry.diff (hunk 1 above)
Comment (jsjoeio) [reply patches/telemetry.diff:107]: Uh...looks like I'm not using one? 😅 Reminds me of when I first started at Coder and Anmol/I were using different algorithms.

image

Okay I will leave as is for now then and merge this!

Comment (code-asher) [reply patches/telemetry.diff:107]: Ah that probably means it is using the default (`myers`).

**Pull request merged**